### PR TITLE
fix: add PAD token

### PR DESCRIPTION
**Pull request by tuanlda78202**
# Bug: Incorrect Mask Generation After Audio Padding

## Description
During code review, we discovered that the mask generation after audio padding is incorrectly implemented. The current code creates masks with all 1s for padded audio. Given the padded audio with the padding value = 0:
```
Input Audio: [1, 2, 3, 4, 5, 0, 0, 0, 0, 0]
```

```python
# Current problematic code
concatenated_audio = self.pad_audio(concatenated_audio)
mask = torch.zeros(30 * 16000 // 320, dtype=torch.bool)
audio_frames = min(len(concatenated_audio), self.max_audio_length) // 320
mask[:audio_frames] = 1  # Bug: This includes padding tokens
```
- The training code reserves a special VQ token for padding (vq_codes + 1).
- However, with mask=1 everywhere, ~mask becomes all zeros:
   So instead of creating a mask: ```Expected Mask: [1, 1, 1, 1, 1, 0, 0, 0, 0, 0] ```, author created a buggy mask: ```[1, 1, 1, 1, 1, 1, 1, 1, 1, 1] ```
- Result: The special padding embedding is never applied cause ~mask will be all 0s.
```python
# Impact on training
if self.training and self.config.mask_embs and mask is not None:
    x[~mask] = project_out(self.rq.layers[0]._codebook.embed[0, self.vq_codes])
```

## Impact
- The padding tokens is not trained.
- Model sees raw embeddings in padding regions instead of consistent padding tokens. Leading to longer semantic token sequence with fixed length = 750 when encode audio.

### CODE REVIEW

**Comment by bachvudinh** [comment]
lgtm!

**Review by bachvudinh — APPROVED** [review]
